src/main/kotlin/no/nav/syfo/melding/database/MeldingQuery.kt

```diff
@@ -53,16 +53,27 @@ const val queryGetMeldingForConversationRefAndArbeidstakerident =
         SELECT *
         FROM MELDING
         WHERE conversation_ref = ? AND arbeidstaker_personident = ? AND NOT innkommende
```

Comment:
Hvorfor må vi egentlig ha personident i denne where-clausen også? `conversations_ref` vil alltid være for én person 🤔 

Man ser feks i `getBehandlerRefForConversation()` så er det allerede hentet ut meldinger for personident, så det er litt rart å sende inn denne på nytt igjen i neste spørring.

Reply:
Godt spørsmål. Av samme grunn man sjekker det i hasSendtMeldingForConversationRefAndArbeidstakerIdent? 🤷‍♂️ 
Det skader vel ikke, men hvis det er overflødig så kan vi kanskje droppe å bruke samme spørring...

Reply:
Ja, men jeg skjønner ikke helt grunnen i `hasSendtMeldingForConversationRefAndArbeidstakerIdent` heller? Har man en conversation_ref, så er jo den unik og burde bare være knyttet til en personident? Vil `AND-en` noen gang gjøre resultatsettet mindre? Eller er det mer en safety-check, at det faktisk er riktig person?

Reply:
Hva tenker du @geir-waagboe ?

Reply:
Ja, jeg tenkte på det som en form for safety-check ja: hvis conversationRef matcher så skal også personident'en være lik.